docs/src/PolyhedralGeometry/Polyhedra/pg_polyhedra_constructions.md

```diff
@@ -0,0 +1,103 @@
+```@meta
+CurrentModule = Oscar
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["pg_polyhedra_constructions.md"]
+```
+
+# Constructions
+
+The standard way to define a polyhedron is by either giving a V-description or an H-description.
+But polyhedra may also be constructed through other means: by name, via operations on other polyhedra, or from other objects in Oscar.
+
+## H and V representations
+
+### Intersecting halfspaces: H-description
+
+Based on the definition of an $H$-representation, the constructor of `Polyhedron` can be used:
```

Comment:
What's the purpose of this kind of filler text? Aren't the docstrings being inserted here plus the headline explanation enough? If not, perhaps those should be improved instead of inserting chit-chat?